### DISCUSSION TITLE
1.0.9.4

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#61] [Обновления]
## 1.0.9.4
- Добавлен перевод для 9GAG. Работает только на видео открытом в отдельной вкладке (видео для теста: https://9gag.com/gag/anzjZzn). При смене видео не воспроизводится аудио перевода (пока не разобрался как это исправить).
- Теперь, старая кнопка загруки первода удаляется при смене видео (twitch, vimeo)
- Добавлен перевод для Vimeo (видео для теста: https://vimeo.com/149640932)
- Уменьшено округление краев у кнопки загрузки
- Элементы абсолютного позиционирования перенесены в единый контейнер

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.0.9.3...1.0.9.4

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.0.9.4'>1.0.9.4</a>.</em>

**Comment by FiLinX** [comment]
хм.. странно, почему то не произошло автообновления сейчас в темпермонке.. 
хотя на 1.0.9.3 когда, - всё на автомате прошло

аа, блин, только после коммента глянул, что всего 21 минуту назад обнова была выкачена) 
а в темпермонке у меня настроено на проверку раз в сутки)

**Comment by sailars** [comment]
На Firefox (106) немного есть проблемы с отображением регулировок громкости, скрин прилагаю. Если нажать на Перевести, то слайдеры появляются. 

![изображение](https://user-images.githubusercontent.com/67970092/205484829-173ac664-c2f4-4c86-b063-b752e2ae33bb.png)

**Reply by ilyhalight** [reply]
> На Firefox (106) немного есть проблемы с отображением регулировок громкости, скрин прилагаю. Если нажать на Перевести, то слайдеры появляются.
> 
> ![изображение](https://user-images.githubusercontent.com/67970092/205484829-173ac664-c2f4-4c86-b063-b752e2ae33bb.png)

Забыл поменять, чтобы при включенной галочке отображался слайдер сразу. С слайдером громкости перевода так и задумано

**Reply by ilyhalight** [reply]
Поменяю в 1.0.9.6, если не забуду

**Comment by SVS696** [comment]
А планируется адаптация на смартфонах? Я загрузил расширение в adguard, кнопка появляется на полной версии сайта, но то работает, то нет. И вроде перевод тоже не подтянулся

**Reply by ilyhalight** [reply]
На данный момент не планируется
